lambda-code/nagware/lib/overdueResponseCache.ts

```diff
@@ -0,0 +1,24 @@
+import { RedisConnector } from "./redisConnector.js";
+
+/**
+ * Handles caching of overdue form responses.  The cache expiry should be at least
+ * as long as the interval between Nagware function runs.
+ * 
+ * This cache is used by the app to reduce the number of DynamoDB calls required
+ * when loading the user's `/forms` page. 
+ */
+
+const OVERDUE_CACHE_EXPIRE_SECONDS = process.env.OVERDUE_CACHE_EXPIRE_SECONDS ?? "259200"; // 3 day default
```

Comment:
It looks like `process.env.OVERDUE_CACHE_EXPIRE_SECONDS` is not set. We could just delete it and use the default value.

Reply:
(in case your goal was to have something configurable for the future)

In our Lambda functions we tend to use Environment variables to pass informations that are dynamically defined through the creation of other Terraform resources. In this case we can just hard code the expiration date.

Reply:
Sounds good!  I'll switch to hard-coded values for those constants and you are exactly right.  My goal had been to make them configurable via env vars, but totally fine going the other way as well.